### DISCUSSION TITLE
PHP 8.1 complains about nulls passed to eg strlen

### DISCUSSION THREAD

**Discussion by doubledrat** [rectorphp/rector#7158] [Q&A] [answered]
but rector doesn't seem to have a fix for this?

e.g. strlen($string) -> strlen($string ?? '')

**Comment by samsonasik** [comment, marked as answer]
it already handled by `NullToStrictStringFuncCallArgRector`, ref https://getrector.org/demo/3258c94a-9879-4a88-8eed-9a1b52ea3e43

**Comment by doubledrat** [comment]
ah.  thanks.  from the rules overview of NullToStrictStringFuncCallArgRector I thought it was only subbing null for ''!
